help/transform/missing-values-imputation.md

```diff
@@ -2,31 +2,22 @@
 title: "Missing values imputation"
 ---
 
-Imputation is the process of replacing missing data with substituted values.
+[Imputation](https://en.wikipedia.org/wiki/Imputation_\(statistics\)) is the process of replacing missing data with substituted values.
 
-## Algorithm
-
-Missing values imputation algorithm is based on **k-nearest neighbors algorithm (k-NN)**. This algorithm is a
-non-parametric method used for classification and regression. Both for classification and regression, it can be useful
-to assign weight to the contributions of the neighbors, so that the nearer neighbors contribute more to the average than
-the more distant ones. For example, a common weighting scheme consists in giving each neighbor a weight of **1/d**,
-where **d** is the distance to the neighbor.
+Datagrok imputes missing values using the k-nearest neighbors method ([k-NN](https://public.datagrok.ai/js/samples/domains/data-science/missing-values-imputation)).
 
 ## Run
 
-* Open table
-* Run from menu: **Tools** | **Data Science** | **Missing Values Imputation...**
-* Select source table
-* Select columns that contains missing values **"impute"**
-* Select data rows **"data"**
-* Set **"Number of nearest neighbors"**
-* Run missing values imputation. Result will replace all missing values in all columns and rows.
+* Open a table
```

Comment:
All sequences should use numbered lists. Also, You don't need to mention the first step (open a table) as it is clear from the context.
You don't "run" a top menu"..., you click it with your mouse. Also, follow the location->action sequence. Also. You should specify the outcome/result of each step:
1. On the **Top Menu**, select **ML** > **Missing Values Imputation...**. A dialog opens.
2. In the dialog, specify the columns with missing values (in the **Impute** field) and the columns for imputation (in the **Using** field). You can also select a distance metric, neighbor count, and decide whether to replace missing values or create a new column with imputed results.
3. Click **Run** to execute.